scripts/build/termux_step_massage.sh

```diff
@@ -205,7 +206,8 @@ termux_step_massage() {
 		local nproc=$(nproc)
 		echo "INFO: Identifying files with nproc=${nproc}"
 		local t0=$(get_epoch)
-		local files=$(find . -type f)
+		# recursively search for all names of files (including binaries) containing elf magic at the start of the first line.
+		local files=$(grep -ralm1 '^'"$'\x7fELF'" .)
```

Comment:
- 1.) If this is a list it should be put into an array using `readarray` or `read -a`.
(And adjust the usage of the variable accordingly)
- 2.) Please again do explain the options used here in a comment, or use the long names of the options.

Reply:
Actually I'd like to keep refactoring out of scope of this PR. We can do this other time.
I mean this PR already does a big change/improvement and in the case if something bad happens we will be able at least to bisect to this specific change and fix it without guessing what part of code was broken.

Reply:
At least quote it.
I'll grumble about using a string as a list later.

Reply:
Done.